integrations/prodigy_ner_fashion/requirements.txt

```
@@ -0,0 +1,2 @@
+spacy-streamlit>=1.0.0a0
+streamlit
```

We don't need Streamlit here (see my previous comment about removing the visualisers), but we can add `plotext`, so you can run `train-curve` with `--show-plot`.

```suggestion
plotext
```